docs/tutorials/cpp.md

```diff
@@ -105,24 +109,11 @@ C++ versions beyond C++17 are supported, as are other C++ compilers.
 
 ## Embedded platforms (`no_std`)
 
-Users wishing to use ICU4X on a `no_std` platform will need to write their own crate depending on `icu_capi` that fills in an allocator and a panic hook, similar to what we do in our [example](c-tiny/fixeddecimal/crate/src/lib.rs):
-
-```rust
-#![no_std]
-
-// Expose icu_capi symbols
-extern crate icu_capi;
-
-#[global_allocator]
-static ALLOCATOR: dlmalloc::GlobalDlmalloc = dlmalloc::GlobalDlmalloc;
+Users wishing to use ICU4X on a `no_std` platform will need to provide an allocator and a panic hook in order to build a linkable library. The `icu_capi` crate can provide a looping panic handler, and a `malloc`-backed allocator, under the `panic-handler` and `libc-alloc` features, respectively.
```

Comment:
Question: how does panic-abort and panic-immediate-abort work with this new built-in panic handler?